### PR TITLE
TINY-8207: Added theme registration our mcagar wrapper file. Removed all manual theme registration in tests.

### PR DESCRIPTION
**Pull request by TheSpyder**
Added theme registration our `mcagar` wrapper file.
Removed all manual theme registration in tests.

Related Ticket: TINY-8207

Description of Changes:
* Deleted a lot of code. Feels good.
* search/replace + `eslint --fix` = winning
* Only two significant files to review:
  * `modules/tinymce/src/core/test/ts/module/McAgar.ts`
  * `package.json` where I added an `eslint-fix` script, it's quite handy

Pre-checks:
* [x] ~Changelog entry added~
* [x] Tests have been added (if applicable)
* [x] ~Branch prefixed with `feature/` for new features (if applicable)~

Review:
* [x] ~Milestone set~
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "tsc": "tsc -b",
     "watch": "tsc -b -w",
     "eslint": "lerna exec --stream --no-bail -- eslint src/**/*.ts",
+    "eslint-fix": "yarn -s eslint --fix",
```
Comment (lnewson) [inline package.json:19]: Why is this needed? You can just do `yarn eslint --fix` and it's only 1 or 2 extra characters. This is how all our repos are setup so I'm not keen on adding a custom command for this :shrug:

Comment (TheSpyder) [reply package.json:19]: maybe not everyone knows eslint can fix things for them 🤷‍♂️

File: modules/tinymce/src/core/test/ts/module/McAgar.ts [hunk 2]
```diff
@@ -1,7 +1,10 @@
+/* eslint-disable @tinymce/no-main-module-imports */
 /*
   This is a helper module that allows us to ensure that TinyMCE core is included/compiled
   before mcagar loads. This ensures that the core of the editor isn't loaded from pre-built
   files in node_modules and is always the latest version.
  */
 import 'tinymce';
 export * from '@ephox/mcagar';
+
+import 'tinymce/themes/silver/Main';
```
Comment (lnewson) [inline modules/tinymce/src/core/test/ts/module/McAgar.ts:10]:
```suggestion
import 'tinymce/themes/silver/Main';
export * from '@ephox/mcagar';
```
This should probably be registered before loading the mcagar components. Overall I don't think it'll matter, but this makes more logical sense as it's setting up TinyMCE and then exporting the components to interact with it.

**Review by lnewson — APPROVED** [review]

File: modules/tinymce/src/core/test/ts/module/McAgar.ts (hunk 2 above)
Comment (TheSpyder) [reply modules/tinymce/src/core/test/ts/module/McAgar.ts:10]: Oh interesting. The tests pass (both here and on the model branch), but that makes sense.

File: package.json (hunk 1 above)
Comment (lnewson) [reply package.json]: Maybe we should add something to the Contributing docs section instead then?

**Review by JamesToohey — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (TheSpyder) [reply package.json]: Done, see what you think

Comment (lnewson) [reply package.json]: LGTM :+1:

**Review by ashfordneil — APPROVED** [review]
